remote_states.tf

```diff
@@ -30,7 +44,7 @@ data "terraform_remote_state" "dynamic_assessment" {
     key            = "cool-accounts/dynamic.tfstate"
   }
 
-  workspace = var.assessment_account_name
+  workspace = local.workspace_name
```

Comment:
It's probably worth a brief comment here to explain why this one is different from the others (even though things are well-explained in the comment at the top of this file).

Reply:
This is what threw me off.  Until I expanded the whole file.

Reply:
Good idea.  Comment added in commit f9b0484.